compiler/rustc_target/src/spec/mod.rs

```diff
@@ -165,6 +167,7 @@ flavor_mappings! {
     ((LinkerFlavor::Lld(LldFlavor::Ld64)), "ld64.lld"),
     ((LinkerFlavor::Lld(LldFlavor::Ld)), "ld.lld"),
     ((LinkerFlavor::Lld(LldFlavor::Link)), "lld-link"),
+    ((LinkerFlavor::Lld(LldFlavor::Gcc)), "gcc-lld"),
```

Comment:
Maybe this should be `lld-gcc`.

Reply:
I'm using the name given by @Gankra, though I don't feel particularly strongly about it.